build.gradle

```diff
@@ -20,7 +20,8 @@ ext {
 
   // Third parties
   offheapVersion = '2.2.2'
-  statisticVersion = '1.1.0'
+  managementVersion = '5.0.0.beta'
+  statisticVersion = '1.1-SNAPSHOT'
```

Comment:
Yes there are some other terracotta stat changes.  I had a chat with Anthony and he helped me sort out the details.  I have submitted PR: 
https://github.com/Terracotta-OSS/statistics/pull/10
These changes need to be merged and released.  Once this is done I will update the gradle.build with the new version.  I believe it will be 1.2.0